Extensions/Oxide.Ext.CSharp/CSharpPlugin.cs

```diff
@@ -271,7 +271,10 @@ public virtual void SetPluginInfo(string name, string path)
                    if (!required_plugin)
                     continue;
                     
-                   required_plugin.Loader.Load(Interface.Oxide.PluginDirectory, plugin);
+                   if (!required_plugin.Loader.Load(Interface.Oxide.PluginDirectory, plugin))
+                   {
+                       Interface.Oxide.LogException($"{Name} requires plugin '{plugin}' but it could not be loaded!");
```

Comment:
{Name} requires plugin '{plugin}' but it could not be loaded!
should be:
{Name} requires plugin '{plugin}' and it could not be loaded!